README.md

```diff
@@ -303,6 +311,20 @@ FIL_PROOFS_COLUMN_WRITE_BATCH_SIZE=Y
 
 Note that this value affects the degree of parallelism used when persisting the column tree to disk, and may exhaust system file descriptors if the limit is not adjusted appropriately (e.g. using `ulimit -n`).  If persisting the tree is failing due to a 'bad file descriptor' error, try adjusting this value to something larger (e.g. 524288, or 1048576).  Increasing this value processes larger chunks at once, which results in larger (but fewer) disk writes in parallel.
 
+When the library is built with both CUDA and OpenCL support, you can choose which one to use at run time.  Use the environment variable:
+
+```
+FIL_PROOFS_GPU_FRAMEWORK=cuda
```

Comment:
Setting this value on master currently appears to do nothing -- opencl is used in all cases for me unless I specify `--features cuda` on the command line

Reply:
Yes, it's part of this PR.